composer.lock

```diff
@@ -240,26 +240,26 @@
         },
```

Comment:
will need to revert the change in this file (i usually just cheat when do this and simply copy in the file from master branch, then it will go away on the diff :) )

Reply:
I think the diffs are from running a 'composer update' and having the packages referenced in the lock file increment versions if they are allowed to by composer.json. Do you guys have a standard procedure for updating package dependencies?

Reply:
we rely on [dependabot](https://dependabot.com/#how-it-works)

just kidding i think :)